server/sandbox_run_linux.go

```diff
@@ -712,6 +713,12 @@ func AddCgroupAnnotation(ctx context.Context, g generate.Generator, mountPath, c
 	return cgroupParent, nil
 }
 
+func configureGeneratorWithCollectMode(g generate.Generator) {
+	// we always want inactive-or-failed as the collect mode
+	// when using systemd
```

Comment:
here "when using systemd" does not make sense, because there is no such thing as "collect mode" without systemd